easybuild/easyconfigs/s/sinto/sinto-0.7.3.1-foss-2020b-Python-3.8.6.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'sinto'
+
+version = '0.7.3.1'
+
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
'-Python-%(pyver)s' should no longer be included in versionsuffix (when using the default Python for that toolchain)